homeassistant/components/glances/__init__.py

```diff
@@ -174,6 +171,4 @@ def get_api(hass, entry):
     """Return the api from glances_api."""
     params = entry.copy()
     params.pop(CONF_NAME)
-    verify_ssl = params.pop(CONF_VERIFY_SSL)
-    session = async_get_clientsession(hass, verify_ssl)
-    return Glances(hass.loop, session, **params)
+    return Glances(**params)
```

Comment:
We want the library to use the Home Assistant httpx client.

https://github.com/home-assistant/core/blob/80833aa7fbdcbced372b00dadbfab9e938f98053/homeassistant/helpers/httpx_client.py#L23-L25

Reply:
Do you mean I should also make changes to the library?

Reply:
I think you should discuss it with the code owner of the library yes. Maybe there is a reason why the code owner removed the ability to provide the http client during the migration from aiohttp to httpx. Or maybe it was just forgotten....

Reply:
Looks like the codebase was already migrated to httpx (in release 0.3.0).
Here's the release changelog:

https://github.com/home-assistant-ecosystem/python-glances-api/commit/16e4156153a83f972d1b47cde6dfb19a0a274f85#diff-0ba59922ef343eb7f8956ec7ced35522e21e1875d947fa84c2a82ed57168dce1

Reply:
Yes, but it doesn't explain why @fabaff removed the session argument.

Reply:
I think it's ok to go ahead here without an update to the library, but it would be good to update the library and adjust here again asap.